common/src/test/java/tachyon/security/authentication/SimpleAuthenticationProviderImplTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security.authentication;
+
+import org.junit.Test;
+
+import tachyon.Constants;
+
+public class SimpleAuthenticationProviderImplTest {
+  @Test
+  public void createAnonymousAuthenticationProviderTest() throws Exception {
+    System.setProperty(Constants.TACHYON_AUTHENTICATION_PROVIDER_CUSTOM_CLASS,
+        SimpleAuthenticationProviderImpl.class.getName());
+    new SimpleAuthenticationProviderImpl();
```

Comment:
`SimpleAuthenticationProviderImpl` should allow any user/password combination, how about adding one or two testcase this? e.g., testing "":"" (both empty string),  "foo":"bar" ?